rustfmt.toml

```diff
@@ -0,0 +1,14 @@
+# See https://github.com/rust-lang/rustfmt for more information
+
+# `Foobar { foo, bar }` is more readable than `Foo { foo: foo, bar: bar }`
+use_field_init_shorthand = true
+
+# Forces let else blocks to always be their own line(s)
+single_line_let_else_max_width = 0
+
+# Subjectively, this combo improves readability on imports. The full prefix is always
+# visible right after `use`, and different types of imports are grouped together in
+# a predictable way (first std, then other crates, then local imports).
+# (These options are still unstable and thus commented out)
+# imports_granularity = "Module"
+# group_imports = "StdExternalCrate"
```

Comment:
I agree with both of these values and yes the only reason they haven't been enabled is because they're still unstable. We can have them in the file already though as seen here.